modules/checking-mco-status-certs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="checking-mco-status-certs_{context}"]
+= Viewing and interacting with certificates
+
+The following certificates are handled in the cluster by the Machine Config Controller (MCC) and can be found in the ControllerConfig:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
+Use the following procedure to get information about the certificates, including the underyling bundle the certificate comes from, and the signing and subject data.
+
+.Procedure
+
+* Run the following command to get certificate information:
++
+[source,terminal]
+----
+$ oc describe controllerconfig/machine-config-controller
+----
++
+.Expected output
++
+[source,text]
+----
+"controllerCertificates": [
+                   {
+                       "bundleFile": "KubeAPIServerServingCAData",
+                       "signer": "<signer_data1>",
+                       "subject": "CN=openshift-kube-apiserver-operator_node-system-admin-signer@168909215"
+                   },
+                   {
+                       "bundleFile": "RootCAData",
+                       "signer": "<signer_data2>",
+                       "subject": "CN=root-ca,OU=openshift"
+                   }
+                ]
+----
+
+* Alternatively, you can access a simpler version of the information found in the ControllerConfig by checking the machine config pool status:
++
+[source,terminal]
+----
+$ oc get mcp master -o yaml | grep certExpirys -B1 -A15
+----
++
+.Expected output
++
+[source,text]
+----
+status:
+  certExpirys:
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=admin-kubeconfig-signer,OU=openshift
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kube-csr-signer_@1689585558
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kubelet-signer,OU=openshift
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kube-apiserver-to-kubelet-signer,OU=openshift
+  - bundle: KubeAPIServerServingCAData
+    subject: CN=kube-control-plane-signer,OU=openshift
+----
++
+This method is meant for {product-title} applications that already consume machine config pool information.
+
+You can check which certificates are on the nodes by looking at `/etc/docker/cert.d`.
```

Comment:
Should this be presented as a third bullet option?

```suggestion
* To check which certificates are on the nodes, review the `/etc/docker/cert.d` file.
```

(or directory, not sure which it is)

Reply:
This is a directory, so the we should not change this bullet. I will grab an example of this

Reply:
```
sh-5.1# ls /etc/docker/certs.d
image-registry.openshift-image-registry.svc.cluster.local:5000	image-registry.openshift-image-registry.svc:5000
```
and then
```
sh-5.1# ls /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000
ca.crt
```
so each IR cert has one entry called ca.crt inside of its sub directory

Reply:
I ended up adding it as a separate bullet with an example output. Thanks y'all!